apps/hash-api/src/index.ts

```diff
@@ -283,6 +285,55 @@ const main = async () => {
     next();
   });
 
+  /**
+   * Add a proxy for requests to the Ory Kratos public API, to be consumed
+   * by the frontend for authentication related requests made in the
+   * browser. Note that server-side frontend authentication requests
+   * can be sent the the Ory Kratos public URL directly, because the
+   * CORS requirements are not as strict as the one from the browser.
+   */
+  app.use("/auth/*", cors(CORS_CONFIG), (req, res, next) => {
```

Comment:
@nonparibus @CiaranMn it was my understanding from the ticket we do not want to have the `ory` name in the path for requests made to Ory Kratos in the browser, so I'm using the `/auth/*` prefix for these routes instead of something like `/ory-kratos/*`. 
I do think this is slightly less intuitive for developers to understand that this is a direct proxy to the Ory Kratos API, and not auth endpoints that are maintained by us, but I suppose that's not a huge deal.

Reply:
As a side note, even though this is less intuitive, it also prevents vendor lock-in, as we no longer name by the service that provides it, but instead the function that the underlying proxied service provides.